pkgs/tools/filesystems/upspin/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "upspin";
+  version = "unstable-2023-02-05";
```

Comment:
```suggestion
  version = "2023-02-05";
```

Reply:
This is my mistake, forget it.
See https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming

Reply:
Forgotten.